demo/java-swing/src/Main.java

```diff
@@ -44,6 +45,32 @@
     }
 
     public static void main(String[] args) {
+
+        Options options = BuildCommandLineOptions();
+        CommandLineParser parser = new DefaultParser();
+        HelpFormatter formatter = new HelpFormatter();
+
+        CommandLine cmd;
+        try {
+            cmd = parser.parse(options, args);
+        } catch (ParseException e) {
+            System.out.println(e.getMessage());
+            formatter.printHelp("rhinofiledemo", options);
```

Comment:
rhinofiledemo?

Reply:
I had to create a second branch due to some issue with submodules for this binding and I think fixed these small things on the other branch by mistake :) good catch!